code/game/objects/items/weapons/clown_items.dm

```diff
@@ -50,7 +50,7 @@
 		M.stop_pulling()
 		M << "\blue You slipped on the [name]!"
 		playsound(src.loc, 'sound/misc/slip.ogg', 50, 1, -3)
-		M.Stun(3)
+		M.Stun(4)

```

Comment:
Might want to note that this was increased, rather than decreased as with most other stuns.